esmvaltool/diag_scripts/hydrology/globwat.py

```diff
@@ -0,0 +1,346 @@
+"""Globwat diagnostic.
+
+Currently, the model reads monthly reference evaporation and then multiplies
+it by kc and then divide it by the number of days in each month and uses it
+as daily actual evaporation in the model. We can change the model code to
+read daily reference evaporation and do the calculation, but we
+did not change the model code for this aim yet.
```

Comment:
This seems to be about the model itself, but it doesn't matter for the recipe, does it? Can we leave it out?

Reply:
I deleted this part.